lib/main.js

```diff
@@ -48,7 +49,11 @@ const watchFile = ({
   // if the hash file doesn't exist
   // or if it does and the hash is different
   if (!fs.existsSync(packageHashPath) || fs.readFileSync(packageHashPath, 'utf-8') !== recentDigest) {
-    console.log('package.json has been modified. Installing...')
+    console.log(`package.json has been modified.${isHashOnly ? '' : 'Installing...'}`)
```

Comment:
I think we should have two separate log statements. The original one and one on line 55 that says that it's only going to update the hash.

Reply:
Add another log. Feel free to let me know if you have any idea or you want to change the log.

Reply:
I'm confused. Why is the log on L53 just an empty string? And why is the one on L57 saying we're only going to update the hash when that's not true? I suggest:

L52: `package.json has been modified`
L54: `Updating hash only because --hash-only is true.`
L57: `Installing and updating hash`

Reply:
Oh. Sorry. I forget to push.

Reply:
Done.